internal/table/table.go

```diff
@@ -133,51 +140,120 @@ func rescaleColumns(colWidths []int, targetTotal int) {
 	colWidths[len(colWidths)-1] += targetTotal - mathutils.Total(colWidths...)
 }
 
+func getCroppedText(text string, maxLen int) []entry {
+	entries := make([]entry, 0)
+	pos := 0
+	matches := colorize.ColorRx.FindAllSubmatchIndex([]byte(text), -1)
+	runeText := []rune(text)
+
+	for pos < len(runeText) {
```

Comment:
I feel like we can do this in a single for loop, rather than the two. You're already asserting this conditional on line 159. And the loop on line 152 is basically just cutting things off at the line marker, which you could also do with a conditional.

I think it's probably most sensible to drop the for loop on 152 and just start a new entry when count==maxLen.